pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -176,4 +177,68 @@ public SuccessResponse dropInstance(
     }
     return new SuccessResponse("Successfully dropped instance");
   }
-}
+
+  @GET
+  @Path("/instances/{instanceName}/tags")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Get all the tags for the specified instance", produces = MediaType.APPLICATION_JSON)
+  @ApiResponses(value = {@ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 404, message = "Instance not found"), @ApiResponse(code = 500, message = "Internal error")})
+  public String getInstanceTags(
```

Comment:
IMO `getInstance()` is good enough for this purpose, no need to add another one?